test/test_vim.py

```diff
@@ -54,27 +54,6 @@ def test_strwidth():
     # 19 * 2 (each japanese character occupies two cells)
     eq(vim.strwidth('neovimのデザインかなりまともなのになってる。'), 44)
 
-
-@with_setup(setup=cleanup)
-def test_list_runtime_paths():
-    # Is this the default runtime path list?
-    homedir = os.path.join(os.environ['HOME'], '.nvim')
```

Comment:
Because it is not enough. (And env dependent) `nvim -u NONE` and then `set rtp?` has many more entries.